ui/src/app/kvm/views/KVMList/AddKVM/AddLxd/SelectProjectForm/SelectProjectForm.tsx

```diff
@@ -98,12 +99,11 @@ export const SelectProjectForm = ({ authValues }: Props): JSX.Element => {
           zone: Number(authValues.zone),
         };
         dispatch(podActions.create(params));
-        setSavingPod(authValues.name || "LXD VM host");
       }}
       saved={saved}
       savedRedirect="/kvm"
```

Comment:
Can this prop be used for redirecting? If not should this prop be removed?

Reply:
Oh I forgot about that! I could probably use it instead of having a Redirect component here